.github/workflows/nightly.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Nightly'
+
+on:
+  schedule:
+    - cron: '* 8 * * *' # 8am in UTC is midnight PST
```

Comment:
let's do something like 8am PST so that no one gets their sleep interrupted with email notifications

Reply:
Actually let's do 6am

Reply:
Done.